notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1191 @@
+{
```

Comment:
<p>Does the Brier score work correctly if the probabilities are not normalized? Looks like it is the mean squared difference of the probabilities from their target (either 0.0 or 1.0) So if the output, ypred, is not a probability, then the results of this score will be misleading, right?</p><p> </p><p>https://github.com/scikit-learn/scikit-learn/blob/2beed5584/sklearn/metrics/_classification.py#L2510</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Yes, I totally agree that these metrics are not giving us the results we are looking for because these are just metrics for the correct classification of the labels `pass` or `fail` and not the probability of getting classified into the classes.

In order to get a metric for the probability model, we have created the calibration curve that allows us to choose a better model in this scenario.